### PR TITLE
fix(app): fix software update inline notification layout

### PR DESCRIPTION
**Pull request by koji**
# Overview
fix software update inline notification layout and fix typo
close RQA-1143

# Test Plan
- Go to RobotSettings Dashboard

# Changelog
- update RobotSetting Button component

# Review requests
The current fix is not ideal since using negative to modify position
If you have a better idea, I'd love to hear that.

# Risk assessment
low

### CODE REVIEW

**Comment by jerader** [comment]
I think you need to rebase or something so you don't have the commit history from `internal-release-0.14.0`

File: app/src/pages/OnDeviceDisplay/RobotSettingsDashboard/RobotSettingButton.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ interface RobotSettingButtonProps {
   robotName?: string
   isUpdateAvailable?: boolean
   enabledDevTools?: boolean
-  enabledHistoricOffests?: boolean
+  enabledHistoricOffsets?: boolean
```
Comment (jerader) [inline app/src/pages/OnDeviceDisplay/RobotSettingsDashboard/RobotSettingButton.tsx:48]: womp, good catch!!

File: app/src/pages/OnDeviceDisplay/RobotSettingsDashboard/RobotSettingButton.tsx [hunk 2]
```diff
@@ -163,7 +163,11 @@ export function RobotSettingButton({
           </StyledText>
         </Flex>
       ) : null}
-      <Flex gridGap={SPACING.spacing40} alignItems={ALIGN_CENTER}>
+      <Flex
+        gridGap={SPACING.spacing24}
+        alignItems={ALIGN_CENTER}
+        marginLeft={isUpdateAvailable ? '-18rem' : undefined}
```
Comment (jerader) [inline app/src/pages/OnDeviceDisplay/RobotSettingsDashboard/RobotSettingButton.tsx:169]: hmm i messed around with this a bit and not sure of a good alternative either 🤔  it works as is though so maybe we can refactor later.

**Review by jerader — APPROVED** [review]

File: app/src/pages/OnDeviceDisplay/RobotSettingsDashboard/RobotSettingButton.tsx (hunk 2 above)
Comment (koji) [reply app/src/pages/OnDeviceDisplay/RobotSettingsDashboard/RobotSettingButton.tsx:169]: I think your implementation didn't break the layout. After your PR a couple of us updated this component and unfortunately, we didn't test software inline notification case when we updated.
I just revisited this PR and figured out another better fix. I will close this and open a new PR for this layout issue.

**Comment by koji** [comment]
This issue will be fixed in a following PR with more better implementations

**Pull request closed without merging**